sotodlib/site_pipeline/make_atomic_filterbin_map.py

```diff
@@ -464,12 +464,17 @@ def main(
             singlestream=args.singlestream,
             site=args.site, unit=args.unit,
             use_psd=args.use_psd,
-            wn_label=args.wn_label,
-            atomic_db=args.atomic_db,) for r in run_list]
+            wn_label=args.wn_label,) for r in run_list]
     for future in as_completed_callable(futures):
         L.info('New future as_completed result')
         try:
-            errors, outputs = future.result()
+            errors, outputs, d_ = future.result()
+            if d_ is None:
+                # this means the map failed and the function returned early
+                continue
```

Comment:
Would likely make more sense to check `errors` than the value of `d_`, no? The usual paradigm is:

```
def my_func(x: float):
  errors = []
  if x == 2.0:
    errors.append("x is two!!!")
    return errors, None

  return errors, 2.0 - x

err, val = my_func(4.0)

if err:
  ... Handle error

... Use val
```

Reply:
Why not reverse the condition?

```
if d_ is not None:
    list_infos = [] ...
```

Since there is nothing else happening under that if I think reverting it is better.